.vscode/settings.json

```diff
@@ -22,4 +22,5 @@
   "search.exclude": {
     "crates/turbopack-tests/tests/snapshot/**": true
   },
+  "typescript.tsdk": "node_modules/typescript/lib"
```

Comment:
This ensures that we are using the workspaces' version of VSCode rather than whichever random version you have installed on your machine.

I get it if we want to move this out into its own PR.